opentasks-contract/src/main/java/org/dmfs/tasks/contract/TaskContract.java

```diff
@@ -1056,14 +1056,15 @@ public static Uri getSearchUri(String authority, String query)
 
 
     /**
-     * Instances of a task. At present this table is read only. Currently it contains exactly one entry per task (and task exception), so it's merely a copy of
-     * {@link Tasks}.
+     * A table containing one entry per task instance. This table is writable in order to allow modification of single instances of a task. Write operations to
+     * this table will be converted into operations on overrides and forwarded to the task table.
      * <p>
-     * TODO: Insert all instances of recurring the tasks.
-     * </p>
+     * Note: The {@link #DTSTART}, {@link #DUE} values of instances of recurring tasks represent the actual instance values, i.e. they are different for each
+     * instance ({@link #DURATION} is always {@code null}).
+     * <p>
+     * Also, none of the instances are recurring themselves, so {@link #RRULE}, {@link #RDATE} and {@link #EXDATE} are always {@code null}.
```

Comment:
Would it make sense to consider not using all of the `TaskColumns` in `Instances` here?
Since `DURATION`, `RRULE`, `RDATE`, `EXDATE` are always `null`.
Having something like a `CommonTaskColumns`?

Reply:
no, the instances table should be compatible with the tasks table, i.e. instances are special case of a task. In particular, a projection that works on the task table should work on the instances table as well.

Reply:
ok